src/content/blog/astro-220.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Astro 2.2"
+description: "CDN support for assets • image() schema helper API change • Much more!"
+publishDate: "April 5, 2023"
+authors:
+  - matthew
+coverImage: "/src/content/blog/_images/astro-220/blog-hero-2_2-release.jpg"
+socialImage: "/src/content/blog/_images/astro-220/og-image-2_2-release.jpg"
+lang: "en"
+---
+
+Astro 2.2 is out! Included in this release are:
+
+- **[CDN support for assets](#cdn-support)**: You can now deploy your assets to a CDN external to your site, and Astro will loads from there.
```

Comment:
"loads" doesn't sound right here, I'd change this sentence to explicitly refer back to "assets".
```suggestion
- **[CDN support for assets](#cdn-support)**: You can now deploy your assets to a CDN external to your site, and Astro will load them from there.
```